cbt-bot/cbt_bot_llm.py

```
@@ -51,83 +52,140 @@ def get_memory(self, user_id: str) -> ConversationBufferMemory:
-                    # Redis에서 사용자 메모리 가져오기
-                    pipe.watch(memory_key)
-                    memory_data = pipe.get(memory_key)
+                    logging.info(f"Task 실행 시작: {prompt_type}")
+                    async for chunk in task:
+                        collected_responses[prompt_type] += chunk
+                        if prompt_type == "consultation":
+                            yield (prompt_type, chunk)
+                            await asyncio.sleep(0)
+                except Exception as e:
+                    logging.error(f"❌ {prompt_type} 작업 실행 중 오류 발생: {e}")
+                    raise;
```

Ruff (0.8.2)

SyntaxError: Expected an expression

---

Statement ends with an unnecessary semicolon

(E703)